includes/model/class-application.php

```diff
@@ -241,11 +241,10 @@ public function get_public_key() {
 	 * @return string The User description.
 	 */
 	public function get_summary() {
-		return \wpautop(
-			\wp_kses(
-				\get_bloginfo( 'description' ),
-				'default'
-			)
+		return sprintf(
+			/* translators: %s: Domain of the site */
+			__( 'This is the Application Actor for %s. It&#8217;s a tool account only, please don&#8217;t follow or interact with it.', 'activitypub' ),
```

Comment:
Feels odd to say that, no? Maybe just return an empty summary instead?

Reply:
Why? This user should simply not be followed at all.

Reply:
the problem is, that people do, becaue I have follow requests for that user. we can remove the please do not follow and only add the first part.